src/commons/Reporter.ts

```diff
@@ -250,7 +250,7 @@ export namespace Reporter {
 function prettyMessage(logLevel: string, msg: string): string {
   const dateString: string = getDate();
 
-  return `${dateString} ${currentTestName} ${logLevel} ${msg}`;
+  return `${dateString}${currentTestName !== '' ? ` ${currentTestName} ` : ' '}${logLevel} ${msg}`;
```

Comment:
Can currentTestName be undefined?
Maybe 
```
${currentTestName && currentTestName !== '' ? ` ${currentTestName} ` : ' '}
```
is safer

Reply:
@refael-m it's defined [here](https://github.com/cloudinary/wdio-allure-ts/blob/a188fc6d9616f2e543aaa299aa42173966c2beae/src/commons/Reporter.ts#L11)